ddl/index.go

```diff
@@ -823,7 +823,7 @@ func doReorgWorkForCreateIndex(w *worker, d *ddlCtx, t *meta.Meta, job *model.Jo
 					tryFallbackToTxnMerge(job, err)
 				}
 				ingest.LitBackCtxMgr.Unregister(job.ID)
-				return false, ver, errors.Trace(err)
+				return false, ver, nil
```

Comment:
If `convertAddIdxJob2RollbackJob` returns an error, it will be incorrectly hidden.

Reply:
```suggestion
				return false, ver, errors.Trace(err)
```